matrix/all-in-one.yaml

```diff
@@ -0,0 +1,534 @@
+component: Splunk Distribution of OpenTelemetry .NET
+version: 1.0.2
+dependencies:
+  - link: https://github.com/open-telemetry/opentelemetry-dotnet
```

Comment:
Shouldn't we list OTel .NET NuGet packages as dependencies (not the repository)?

Reply:
Fabrizio, what are your preferences, where the link should go?
Kind of documentation stuff, or downloadable version?

Reply:
Depends on how you envision these links being used. My preference is to provide as much context as possible, so docs or official repo URL. This file is not meant to function as a dependency list for setting up an environment.

Reply:
12ce8cc8e2454821554cafc936865f8388e5fe5d and 022e5a729d2f179883ae837ee956a41f0e7e1a99